charts/mlrun-ce/templates/tdengine/statefulset.yaml

```diff
@@ -0,0 +1,149 @@
+          ports: {{ range $idx, $val := .Values.tdengine.service.ports.tcp }}
+          - name: tcp{{- $idx}}
+            containerPort: {{ $val }}
+            protocol: TCP
+          {{- end }}
+          {{ range $idx, $val := .Values.tdengine.service.ports.udp }}
+          - name: udp {{- $idx}}
+            containerPort: {{ $val }}
+            protocol: UDP
+          {{- end }}
+
+          env:
+          # POD_NAME for FQDN config
+          - name: POD_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+          # SERVICE_NAME and NAMESPACE for fqdn resolve
+          - name: SERVICE_NAME
+            value: {{ include "mlrun-ce.tdengine.fullname" . }}
+          - name: STS_NAME
+            value: {{ include "mlrun-ce.tdengine.fullname" . }}
+          - name: STS_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          # TZ for timezone settings, we recommend to always set it.
+          - name: TZ
+            value: {{ .Values.tdengine.timezone }}
+          # TAOS_ prefix will configured in taos.cfg, strip prefix and camelCase.
+          - name: TAOS_SERVER_PORT
+            value: "6030"
+          # Must set if you want a cluster.
+          - name: TAOS_FIRST_EP
+            value: '$(STS_NAME)-0.$(SERVICE_NAME).$(STS_NAMESPACE).svc.{{ .Values.clusterDomainSuffix | default "cluster.local" }}:$(TAOS_SERVER_PORT)'
+          # TAOS_FQND should always be setted in k8s env.
+          - name: TAOS_FQDN
+            value: '$(POD_NAME).$(SERVICE_NAME).$(STS_NAMESPACE).svc.{{ .Values.clusterDomainSuffix | default "cluster.local" }}'
+
+          envFrom:
+          - configMapRef:
+              name: {{ include "mlrun-ce.tdengine.fullname" . }}-taoscfg
+          volumeMounts:
+          - name: {{ include "mlrun-ce.tdengine.fullname" . }}-taosdata
+            mountPath: /var/lib/taos
+          - name: {{ include "mlrun-ce.tdengine.fullname" . }}-taoslog
+            mountPath: /var/log/taos
+          readinessProbe:
+            exec:
+              command:
+              - taos-check
+            initialDelaySeconds: 5
+            timeoutSeconds: 5000
+          livenessProbe:
+            exec:
+              command:
+              - taos-check
+            initialDelaySeconds: 15
+            periodSeconds: 20
+          securityContext:
+            # privileged: true
+            # allowPrivilegeEscalation: true
+            # runAsUser: 0
+            # runAsGroup: 0
+            # readOnlyRootFilesystem: false
+            # allowedCapabilities:
+            # - CAP_SYS_ADMIN
+            # - CHOWN
+            # - DAC_OVERRIDE
+            # - SETGID
+            # - SETUID
+            # - NET_BIND_SERVICE
+            # AllowedHostPaths:
+            # - pathPrefix: "/proc"
+            #   readOnly: true # 仅允许只读模式挂载
+            # - pathPrefix: "/sys"
+            #   readOnly: true # 仅允许只读模式挂载
```

Comment:
Lets remove commented out configuration - especially the irrelevant unicode characters.
Also `securityContext` might need to be configured in the values to make sure we don't run as root (as the comment suggests is the default)